mappings/net/minecraft/Bootstrap.mapping

```diff
@@ -11,4 +11,4 @@ CLASS net/minecraft/class_2966 net/minecraft/Bootstrap
 	METHOD method_12851 initialize ()V
 	METHOD method_12852 setOutputStreams ()V
 	METHOD method_17597 getMissingTranslations ()Ljava/util/Set;
-	METHOD method_17598 logMissingTranslations ()V
+	METHOD method_17598 logMissing ()V
```

Comment:
Not only translations I guess?

Reply:
yes, now it calls [`DefaultAttributeRegistry.checkMissing()`](https://github.com/FabricMC/yarn/pull/1256/files#diff-8355421ab6feca78702e0ae01b0db30aR4)